apis/abortion_policy_api.py

```diff
@@ -1,31 +1,34 @@
-# dataset courtesy of Abortion Policy API (https://www.abortionpolicyapi.com/)
+# Dataset courtesy of Abortion Policy API (https://www.abortionpolicyapi.com/)
 
 import requests
 import os
 import json
+import re
 
 APIKEY = os.environ.get("ABORTION_POLICY_API_KEY")
 HEADERS = {"token": APIKEY}
+TYPE_DEFAULTS = {"str": None, "bool": False, "int": 0, "float": 0.0}
 
 
-def get_states(filepath):
+def main():
     """
-    Transfer .txt of states to a list
-
-    Inputs:
-        filepath (string): file path of the states data
+    Retrieves API data from abortionpolicyapi.org, cleans dataset, and exports
+        cleaned data into .json files.
     """
 
-    states = []
+    # get API data
+    policy_areas = get_api_data()
 
-    with open(filepath, "r") as f:
-        for state in f:
-            states.append(state.strip())
+    # fill in missing data
+    for state_policies in policy_areas:
+        fill_in_missing_data(state_policies)
 
-    return states
+    # convert dataset to json
```

Comment:
I think it'd be good to standardize how we do comments. Start with capital letters on them all, etc.